### PR TITLE
KIALI-1541 ServiceGraph to Graph (terminology refactor)

### PR DESCRIPTION
**Pull request by jshaughn**
Sorry in advance for any PRs that get conflicts...

### CODE REVIEW

**Comment by rhqci** [comment]
:heavy_check_mark: Jenkins CI: kiali-ui-pr-e2e-test [#34](https://paste.fedoraproject.org/paste/M-X-8tF0OUfhxmkeUf8maQ)
  * :heavy_check_mark: Istio-and-Kiali-on-Openshift [#[1223]](https://paste.fedoraproject.org/paste/Rv~HCu2gpjLbMpVVRzhQsQ)
  * :heavy_check_mark: Kiali-on-Openshift [#[571]](https://paste.fedoraproject.org/paste/VQRrxLYzeGnhHRrwgzniBQ)
  * :heavy_check_mark: Kiali-Istio-Checker [#[1005]](https://paste.fedoraproject.org/paste/qN8xvasTFgs3X-K7sdGVtw)
  * :heavy_check_mark: run-kiali-e2e-tests [#[55]](https://paste.fedoraproject.org/paste/xozntn65kR1ZtrpAtdmRuA)


KIALI_NAME=kialiuipr666-istio-system

**Comment by mtho11** [comment]
Well, I hope nobody else was working on anything ;)

**Comment by mtho11** [comment]
This might leave us in a state where we can't backport anything to TP?

**Comment by jshaughn** [comment]
I suppose a backport could be an issue as we'd have to move content as opposed to perhaps just a cherry pick.  But I think we need to move forward with this because the terminology is confusing and verbose and we can't worry too much about TP, I think.

File: src/__itests__/App.browser.test.tsx [hunk 1]
```diff
@@ -17,7 +17,7 @@ const getPath = (key: string) => {
 };
 
 const kialiPages = {
-  graph: `${getUrl()}/console/service-graph/all`,
+  graph: `${getUrl()}/console/graph/all`,
```
Comment (mtho11) [inline src/__itests__/App.browser.test.tsx]: Is 'Graph' really better terminology than ' Service Graph'?

Graph is general, Service-graph is specific. What happens when we add other graphs?

**Pull request merged**